specs/api_use_enabled_attribute_from_definitions.adoc

```diff
@@ -0,0 +1,104 @@
+// vim: tw=79
+
+= Use 'enabled' attribute in objects to enable GET API requests
+
+Tendrl API should use the 'enabled' attribute defined in the definitions yaml
+files to enable the API for the specific object.
+
+== Problem description
+
+Tendrl API currently allows GET requests for all available objects, which is
+not the desired expectation, not all objects defined in the definitions files
+need to have a GET API call. 
+
+== Use Cases
+
+Using the configuration from the definitions files help us to limit and control
+what the end user can access via the API.
+
+== Proposed change
+
+Tendrl API should use the 'enabled' attribute defined against each object to
```

Comment:
"enabled" is defined for Flows as well, the flag should be respected while generating POST/PUT to these flows